tests/args_parser.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright (c) 2020 Kamal Heib <kamalheib1@gmail.com>, Inc. All rights reserved.  See COPYING file
+
+import argparse
+import sys
+
+
+class ArgsParser(object):
+    def __init__(self):
+        self.args = None
+
+    def get_args(self):
+        return self.args
+
+    def parse_args(self):
+        parser = argparse.ArgumentParser()
+        parser.add_argument('--dev_name',
```

Comment:
nit: consider using a `-` separator (i.e `--dev-name`)

Reply:
I'm second to that request, or simply "--dev".

Thanks

Reply:
I agree I'll change it to --dev in v3.